_sass/components/popovers/popover-search.scss

```diff
@@ -0,0 +1,80 @@
+.popover-search {
+  background: $white;
+  border: 0;
+  bottom: 0;
+  display: none;
+  height: auto;
+  max-height: 349px;
+  overflow-x: hidden;
+  overflow-y: scroll;
+  pointer-events: all;
+  position: relative;
+  width: 100%;
+
+  &:not(.lunr-hidden) {
+    display: block;
+    visibility: visible;
+  }
+
+  .result {
+    border-bottom: 1px solid $gray-lighter;
+    transition: background .075s ease-in-out;
+
+    &:hover {
+      background: rgba($gray-lighterest, .6);
+    }
+
+    &:active {
+      background: rgba($gray-lighterest, .9);
+    }
+
+    &:last-of-type {
+      border-bottom: none;
+    }
+
+    .a {
+      display: block;
+      padding: $sm;
+    }
+  }
+
+  .result-length {
+    border-bottom: 1px solid $gray-lighter;
+    font-size: 13px;
+    padding: $sm;
+  }
+
+  .post-title {
+    color: $gray-dark;
+    font-size: 14px;
+    font-weight: $weight-bold;
+    line-height: 20px;
+  }
+
+  .post-excerpt {
+    color: $gray-dark;
+    font-size: 12px;
+  }
+}
+
+.popover-search-wrapper {
+  box-shadow: 0 2px 9px rgba($black, .075);
+  max-width: 228px;
+  opacity: 1;
+  overflow-y: auto;
+  position: absolute;
+  top: 80px;
+  transform: scale3d(1, 1, 1) translate3d(-50%, 0, 0);
+  transition:  transform .1s ease-in-out;
+  visibility: visible;
+
+  &.popover {
+    padding: 0;
+  }
+
+  @include media(xs) {
+    max-width: 312px;
+    max-height: 238px;
+    top: 65px;
+  }
```

Comment:
I'm not a fan of how this looks at even smaller mobile sizes (iPhone SE). Can we try increasing the width of the search input to be the same as the navigation links, and also try making the search a popover like it is on larger sizes?

<img width="331" alt="screen shot 2017-05-29 at 6 18 11 pm" src="https://cloud.githubusercontent.com/assets/5341618/26564895/3eb29132-449b-11e7-97fb-ef5c721c8fa2.png">